src/main/java/seedu/address/logic/commands/ExpenseCommand.java

```diff
@@ -130,7 +140,27 @@ public CommandResult execute(Model model) throws CommandException {
             model.addActivity(activity);
         }
 
-        return new CommandResult(String.format(MESSAGE_SUCCESS, persons.size(), successMessage.toString()));
+        return new CommandResult(String.format(MESSAGE_SUCCESS,
+                payingPerson.getName(), amount, description, successMessage.toString()));
```

Comment:
The arguments `payingPerson.getName()` and `amount` are flipped - currently a successful command shows the success message `Expense of Alex Yeoh by 25.00 successfully created'.

Reply:
ohshit